### ISSUE TITLE
A problem with running tests on 1.3.70 (Error: Timeout of 2000ms exceeded.)

### ISSUE THREAD

**Issue by night-crawler** [kotest/kotest#1262]
I have added these packages:

```
implementation("io.kotest:kotest-core:4.0.0-BETA1")
implementation("io.kotest:kotest-core:4.0.0-BETA1")
implementation("io.kotest:kotest-assertions:4.0.0-BETA1")
```

Then I'm trying to run a sample test code from the example (I skipped mocha installation & copying of js sources because they seem to have been installed already by the new js plugin):

```
org.example.SsnTest.invalid ssn FAILED
    Error: Timeout of 2000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves.
```

Here's the code to reproduce the issue:
https://github.com/night-crawler/kotest-issue

Thank you.

**Comment by sksamuel** [comment]
I've not tested kotest 4 with 1.3.70s new js stuff yet. Now that 1.3.70
final is out we'll release beta2 shortly.

**Comment by sksamuel** [comment]
Beta2 is released with 1.3.70 support.

**Comment by eugenio1590** [comment]
I get the same problem with the version of 4.0.7 of the kotest-core-js library and Kotlin/JS

I tried with the version of 4.1.1 but I got an error about the assertions library: 
`Unresolved reference: matchers`

So, I added "io.kotest:kotest-assertions-js" dependency but with version "4.0.7" and I get the original error: 
`Error: Timeout of 2000ms exceeded.`
